builds/dengue/dengue_titers.py

```diff
@@ -141,22 +146,34 @@ def titer_model(process, sanofi_strain = None, **kwargs):
 
 def titer_export(process):
     from base.io_util import write_json
+    from itertools import chain
+    import pandas as pd
+
     prefix = process.config["output"]["auspice"]+'/'+process.info["prefix"]+'_'
+
     if hasattr(process, 'titer_tree'):
         # export the raw titers
         data = process.titer_tree.compile_titers()
         write_json(data, prefix+'titers.json', indent=1)
-        # export the tree model (avidities and potencies only)
+        # export the tree model
         tree_model = {'potency':process.titer_tree.compile_potencies(),
                       'avidity':process.titer_tree.compile_virus_effects(),
                       'dTiter':{n.clade:n.dTiter for n in process.tree.tree.find_clades() if n.dTiter>1e-6}}
         write_json(tree_model, prefix+'tree_model.json')
 
-        validation_values = {'actual': [v[0] for v in process.titer_tree.validation.values()],
-                             'predicted': [v[1] for v in process.titer_tree.validation.values()]}
-        import pandas as pd
-        validation_values = pd.DataFrame(validation_values)
-        validation_values.to_csv(prefix+'tree_model_validation.csv')
+        # export model performance on test set
```

Comment:
The pandas code seems perfect for writing to CSV, but it could live on the documentation website has part of a short tutorial on using the titer model if we want to remove pandas as an augur dependency. (I would be happy to help write said tutorial!)

Reply:
I think that'd be an awesome addition to the documentation, thanks John :)